self-healing/cron.ts

```diff
@@ -63,28 +65,21 @@ async function loadStreamIntoDumpGraph(stream: string): Promise<void> {
   }
 }
 
-async function fetchPage(url: string, page: number): Promise<string | null> {
-  const fullUrl = `${url}/${page}`;
-  console.log(`Loading LDES page ${fullUrl}`);
-
-  const response = await fetch(fullUrl, {
-    headers: {
-      Accept: "text/turtle",
-    },
-  });
-  if (response.status === 404) {
-    console.log(`Page ${page - 1} was the last page of the stream`);
-    return null;
-  }
-
-  if (!response.ok) {
+async function fetchPage(stream: string, page: number): Promise<string | null> {
+  console.log(`Loading LDES page ${page}`);
+  try {
+    const response = await getNode(ldesProducerConfig, {
+      folder: stream,
+      contentType: "text/turtle",
+      nodeId: page,
+    });
+    return await text(response.stream);
+  } catch (e) {
```

Comment:
Need to accept that if the file is not there, the stream is likely just done.
```suggestion
  } catch (e) {
      if (e.status === 404) {
        console.log(
        `Page ${page} not found, assuming it's the last page of the stream`
        );
        return null;
      }
```

Reply:
oops i missed that was a sufggestion, I've changed it manually